### DISCUSSION TITLE
storage/mount-cfs-with-v3

### DISCUSSION THREAD

**Discussion by giscus** [imroc/tke-guide#14] [General]
# storage/mount-cfs-with-v3

背景

https://imroc.cc/tke/storage/mount-cfs-with-v3

**Comment by ermazi** [comment]
"不稳定"具体是指?
测试过, 好像会A写入成功, B没法立马看到的问题

**Reply by imroc** [reply]
那是比较早期的时候, 现在估计已经没问题了, 我记得是偶现挂载报错

**Reply by ermazi** [reply]
记得是今年测试的
